### PR TITLE
Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by fliperama00**

### CODE REVIEW

File: po/LINGUAS [hunk 1]
```diff
@@ -4,3 +4,4 @@
 en
 nl
 fr
+pt
```
Comment (andyholmes) [inline po/LINGUAS:7]: Just make this `pt_BR` and then it looks good to me :slightly_smiling_face:

**Review by andyholmes — CHANGES_REQUESTED** [review]

**Comment by fliperama00** [comment]
Should I just use the --fixup flag here or there is a better way?

**Comment by andyholmes** [comment]
> Should I just use the --fixup flag here or is there a better way?

Just rename the file (`pt_BR.po`) and then you can squash to single commit.

**Comment by fliperama00** [comment]
Amazing. Thank you

**Review by andyholmes — APPROVED** [review]

**Pull request merged**